tests/alignment_test_cases.json

```diff
@@ -97,22 +97,23 @@
             ["{ius orbis terrarum et uni}", "6---------------------------67---"],
             ["ver-", "kjhg--"],
             ["si", "hghg-gF--"],
-            ["qui", "gf---"],
-            ["ha-", "fd---"],
-            ["bi-", "efe--"],
-            ["tant", "f--fh---"],
+            ["","gf---"],
+            ["qui", "fd---"],
+            ["ha-", "efe--"],
+            ["bi-", "f--"],
+            ["tant", "fh---"],
             ["{in eo}", "6------67---"],
             ["", "4"]
         ]
     },
     {
-        "case_name": "Test alignment: Missing music encoded with too few internal hyphens",
-        "text_input": "in {diebus add char} e- {ius} obsessa est",
+        "case_name": "Test alignment: Missing music encoded without proper number of internal hyphens",
```

Comment:
would it make sense to have one test case for too many internal hyphens and one for too few, rather than a single test for an improper number?

Reply:
Good point!